Tasks/JavaToolInstallerV0/package.json

```diff
@@ -23,12 +23,16 @@
     },
     "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
     "dependencies": {
-        "@types/node": "^6.0.101",
+        "@types/mocha": "^5.2.7",
+        "@types/node": "^10.17.0",
+        "azure-pipelines-task-lib": "^3.0.6-preview.0",
         "@types/q": "^1.0.7",
         "azp-tasks-az-blobstorage-provider-v2": "1.0.0",
```

Comment:
I think we should prepare a new Node 10 ready preview version of this package first.